Fix:
error: implicitly declaring library function 'strcpy' with type 'char *(char *, const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strcat' with type 'char *(char *, const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'system' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'chmod' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- unzoo.c.orig	2002-05-06 10:34:00.000000000 -0500
+++ unzoo.c	2022-01-05 07:21:42.000000000 -0600
@@ -244,6 +244,11 @@
 *H
 */
 #include        <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <utime.h>
 
 
 /****************************************************************************
